services/drupal/composer.json

```diff
@@ -427,7 +427,8 @@
         ],
         "post-install-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createFilesDir",
-            "@install-codestandards"
+            "@install-codestandards",
+            "rm web/web.config"
```

Comment:
@gustavderdrache Do you know if the web-root will always be "web"? Is me hard-coding "web/web.config" here incorrect?

Reply:
We've hardcoded the `web/` directory in a bunch of other configurations, so you're completely fine hardcoding it here.